tests/system/model_monitoring/test_model_monitoring.py

```diff
@@ -332,6 +332,11 @@ def test_model_monitoring_voting_ensemble(self):
         )
         stat = mlrun.get_run_db().get_builder_status(base_runtime)
 
+        # Wait 20 sec if model monitoring stream function is still in building process
+        if base_runtime.status.state in ("building", "waitingForResourceConfiguration"):
+            sleep(20)
+            stat = mlrun.get_run_db().get_builder_status(base_runtime)
```

Comment:
I don't know if you are familiar that we have `retry_until_successful` I think it's better that you'll use that and will set the timeout for that, so that if it finished earlier we won't waste time on the static sleep, remember that in the system tests this is very important to run them as efficient as possible as they are already very long.